crates/types/Cargo.toml

```diff
@@ -12,5 +12,7 @@ serde_json.workspace = true
 thiserror.workspace = true
 uuid.workspace = true
 
+http = "1.1.0"
```

Comment:
can we setup it on the workspace level?

Reply:
As far as I know workspace shall not contain all project dependencies, only most reused one. That will help version management in case any issue will be encountered.